### DISCUSSION TITLE
Air bubble inside meter

### DISCUSSION THREAD

**Discussion by rizz360** [jomjol/AI-on-the-edge-device#1981] [Q&A]
Does anyone have an idea, how I could get rid of the air bubble inside the meter?  
We received a new water meter and it came with an air bubble inside. I had set up my previous meter perfectly, but now with this one the values I get for the second and third digit are all over the place.

![image](https://user-images.githubusercontent.com/1228549/216810660-f11d5a23-b608-40e5-a2fd-96eae1d6774f.png)

**Comment by friedpa** [comment]
@rizz360 I had the same issue, just slidely turn the watermeter by one or two degreeds and the air bubble moves up or down

**Reply by caco3** [reply]
https://www.hausjournal.net/wasseruhr-kondenswasser
